bobtimus/src/http.rs

```diff
@@ -48,17 +48,22 @@ where
 
 async fn create_swap<R, RS>(
     mut bobtimus: Bobtimus<R, RS>,
-    payload: CreateSwapPayload,
+    payload: serde_json::Value,
```

Comment:
Let's open a technical-debt ticket about this.

Reply:
Is this accurate? https://github.com/comit-network/droplet/issues/83

Reply:
Yes